service/javadsl/kafka/server/src/test/scala/com/lightbend/lagom/internal/javadsl/broker/kafka/JavadslKafkaApiSpec.scala

```diff
@@ -106,7 +120,19 @@ class JavadslKafkaApiSpec
 
   override def afterAll(): Unit = {
     application.stop().futureValue
-    kafkaServer.stop()
+
+    kafkaServer.foreach { process =>
+      try {
+        process.destroy()
```

Comment:
I wonder if it makes sense to have some logging here regarding the exit code for the process.

WDYT?

Reply:
There's no exit code here as the process is being killed (destroyed).

This code is from https://github.com/lagom/lagom/blob/1.6.0-M3/dev/build-tool-support/src/main/scala/com/lightbend/lagom/dev/Servers.scala#L65